### ISSUE TITLE
error[E0277]: `cargo_manifest::Product` is not an iterator

### ISSUE THREAD

**Issue by powpingdone** [andrewbaxter/genemichaels#86]
When `cargo install`ing genemichaels (no `--locked`), there are multiple compile errors:
```rust
error[E0277]: `cargo_manifest::Product` is not an iterator
    --> src/bin/genemichaels.rs:248:61
     |
248  |                         for bin in manifest.bin.into_iter().flatten() {
     |                                                             ^^^^^^^ `cargo_manifest::Product` is not an iterator
     |
     = help: the trait `Iterator` is not implemented for `cargo_manifest::Product`, which is required by `cargo_manifest::Product: IntoIterator`
     = note: required for `cargo_manifest::Product` to implement `IntoIterator`
note: required by a bound in `flatten`
    --> /var/home/powpingdone/Devel/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs:1545:21
     |
1542 |     fn flatten(self) -> Flatten<Self>
     |        ------- required by a bound in this associated function
...
1545 |         Self::Item: IntoIterator,
     |                     ^^^^^^^^^^^^ required by this bound in `Iterator::flatten`
```
This might be due to https://github.com/LukeMathWalker/cargo-manifest/commit/0a3a687e98b840d2a0220cbd9f7b75de3cf2e63f but I'm not sure. I had trouble reproducing this at all, and it kinda seemed like removing the `.flatten()` from all of the manifest properties seems to allow compilation.

**Comment by andrewbaxter** [comment]
Oh, thanks for the report! That sounds likely... sorry, it looks like I was too optimistic with the version restrictions on that crate.  I can update this later today.  I think updating that dep and the code, then adding a minor-version restriction to the dep should be good.

**Comment by andrewbaxter** [comment]
Fix released in `0.3.2`
